pkgs/tools/typesetting/xmlto/default.nix

```diff
@@ -23,7 +26,7 @@ stdenv.mkDerivation rec {
 
   # `libxml2' provides `xmllint', needed at build-time and run-time.
   # `libxslt' provides `xsltproc', used by `xmlto' at run-time.
-  nativeBuildInputs = [ makeWrapper getopt ];
+  nativeBuildInputs = [ makeWrapper flex getopt ];
```

Comment:
Is `flex` here mandatory?

Reply:
The patch changes a `.l` file, so flex needs to be run again.

This issue is actually fixed upstream, but there hasn’t been a release. I’ve pushed an updated PR that uses the commits from upstream instead of vendoring the patch.

Reply:
I also added a comment explaining why `flex` is required.